File: CodeForces/579A. Raising Bacteria/Raising Bacteria.py
# Solution by : Tameem Alaa El-Deen Sayed

n = int(input())
c= 0
while n >= 1 :
    if n % 2 == 0 :
        n = n /2
    else:
        n =  n -1 
        n = n / 2 
        c = c + 1 
print (c)

File: CodeForces/579A. Raising Bacteria/Raising_Bacteria.cpp
// Solution By: Tameem Alaa El-Deen Sayed Bakr

#include <iostream>
using namespace std;
 
int main()
{
    int c=0;
    int n ;
    cin >> n ;
 
    while(n>=1)
    {
        if (n % 2 == 1 )
        {
            c++;
            n=n-1;
            n = n / 2 ;
        }
        else
           { n = n / 2 ;}
         
       
    }
    cout << c;
    return 0;
}